bindings/wallet-cordova/src/ios/WalletPlugin.m

```diff
@@ -146,6 +146,41 @@ - (void)WALLET_SET_STATE:(CDVInvokedUrlCommand*)command
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void)WALLET_VOTE:(CDVInvokedUrlCommand*)command
+{
+    CDVPluginResult* pluginResult = nil;
+
+    NSString* wallet_ptr_raw = [command.arguments objectAtIndex:0];
+    NSString* settings_ptr_raw = [command.arguments objectAtIndex:1];
+    NSString* proposal_ptr_raw = [command.arguments objectAtIndex:2];
+    NSString* choice_raw = [command.arguments objectAtIndex:3];
+
+    WalletPtr wallet_ptr = (WalletPtr)[wallet_ptr_raw longLongValue];
+    SettingsPtr settings_ptr = (SettingsPtr)[settings_ptr_raw longLongValue];
+    ProposalPtr proposal_ptr = (ProposalPtr)[proposal_ptr_raw longLongValue];
+    uint8_t choice = (uint8_t)[choice_raw intValue];
+
+    uint8_t* transaction_out = nil;
+    uintptr_t len_out;
+
+    ErrorPtr result = iohk_jormungandr_wallet_vote_cast(wallet_ptr,
```

Comment:
are we deleting/freeing the transaction buffer?

Reply:
Huh, you are right. Missed that. Thanks.